src/esp/gfx/GenericDrawable.cpp

```diff
@@ -54,14 +55,26 @@ void GenericDrawable::draw(const Mn::Matrix4& transformationMatrix,
     lightColors.emplace_back((*lightSetup_)[i].color);
   }
 
+  bool usingDrawableId = false;
+  {
+    RenderCamera& cam = static_cast<RenderCamera&>(camera);
+    usingDrawableId = cam.isRenderingForObjectPicking();
+  }
```

Comment:
why not just a single expression for such a thing?

```suggestion
  bool usingDrawableId = static_cast<RenderCamera&>(camera).isRenderingForObjectPicking();
```